pkg/sfu/buffer/rtpstats_base.go

```diff
@@ -189,15 +205,15 @@ type rtpStatsBase struct {
 	srNewest *RTCPSenderReportData
 
 	nextSnapshotID uint32
-	snapshots      map[uint32]*snapshot
+	snapshots      []snapshot
 }
 
 func newRTPStatsBase(params RTPStatsParams) *rtpStatsBase {
 	return &rtpStatsBase{
 		params:         params,
 		logger:         params.Logger,
 		nextSnapshotID: cFirstSnapshotID,
-		snapshots:      make(map[uint32]*snapshot),
+		snapshots:      make([]snapshot, 2),
```

Comment:
current max usage is two snap shots. So, initialising with that number. It will get recreated if more are added.